test/fix-604.test.js

```diff
@@ -20,5 +20,5 @@ const render = fjs(schema)
 try {
   render(input)
 } catch (err) {
-  t.equal(err.message, 'The value "phone" cannot be converted to a number.')
+  t.equal(err.message, 'The value cannot be converted to a number.')
```

Comment:
This is a completelly useless error.

Reply:
actually avoids a potential reflection attack.

Reply:
After casting or parsing, I have used the same variable to avoid allocating a new one. However, the issue is that the original value is lost and message can't be specific. This is a side effect of my PR, if is a problem, we can close! :(

Reply:
I like the flow changes and the use of language syntax rather than function calls (eg, `+` instead of `Number()`)

Since we're micro optimizing, feels like it should still be a little faster even with the allocation

Can you not add it back and see if there is a gain?

Reply:
On my PC the difference is negligible

Reply:
> actually avoids a potential reflection attack.

also a XSS if the error is send into the response

Reply:
@ivan-tymoshenko @gurgunday  I have reverted the error message https://github.com/fastify/fast-json-stringify/pull/698/commits/ba9de7d0769f98c46e3ea273e49d65e4406fe56a